GoogleForCreators/web-stories-wp
# Editor: Font sizes now allow decimals and updated text presets font sizes.

## Summary

This is a small PR that enables float values for font-size in the Editor.  It also updates the text presets to make sure the font-size of each preset matches the figma docs requirements.

## User-facing changes

- The font-size for each text preset matches the [figma docs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=710%3A0).
- Text blocks now allow for decimal points in font-size:
<img src="https://user-images.githubusercontent.com/40646372/94760643-ef7e4500-0357-11eb-983c-c2c0319f90e4.png" height="300">
NOTE: I added a prop for setting `minWidth` on the dropdown so I could shrink the font-weight dropdown a little bit so fonts with decimal points won't get clipped.

## Testing Instructions

Decimal font-size:
1.) Open up a story in the Editor.
2.) Click on existing text block or add a new text block.
3.) Change the font size to 12.5 or some other decimal'd number and it should apply like normal.

Text Presets:
Add each preset and check that:
"Heading 1" has font-size of `36`
"Heading 2" has font-size of `27`
"Heading 3" has font-size of `21.6`
"Paragraph" has font-size of `18`
"Caption" has font-size of `13.5`
"Overline" has font-size of `12`

Font-size chart from figma design doc:
<img src="https://user-images.githubusercontent.com/40646372/94846651-57737080-03d6-11eb-9d00-85d667596d8d.png" height="300">


Fixes #4419
